ocamltest/ocamltest_stdlib_stubs.c

```diff
@@ -42,6 +42,40 @@
 #include <process.h>
 #include <sys/types.h>
 
+// Developer Mode allows the creation of symlinks without elevation - see
+// https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createsymboliclinkw
+static BOOL IsDeveloperModeEnabled()
```

Comment:
This is not related to the present PR, but for the record this duplication between `ocamltest` and `unix` is not very nice. I see two options:
- Remove the `has_symlink` stuff from `ocamltest`: it is used in a single test [`lib-unix/win-symlink/test.ml`](https://github.com/ocaml/ocaml/blob/9e09fde7350d1afc8db6b2a5d2625fb2a0b8d21d/testsuite/tests/lib-unix/win-symlink/test.ml) and that test could use the `Unix.has_symlink` directly.
- Allow `ocamltest` to depend on `unix`